aiohttp/web_protocol.py

```diff
@@ -523,24 +523,28 @@ def handle_error(self,
         information. It always closes current connection."""
         self.log_exception("Error handling request", exc_info=exc)
 
+        ct = 'text/plain'
         if status == 500:
-            msg = "<h1>500 Internal Server Error</h1>"
-            if self.debug:
-                with suppress(Exception):
-                    tb = traceback.format_exc()
-                    tb = html_escape(tb)
-                    msg += '<br><h2>Traceback:</h2>\n<pre>'
-                    msg += tb
-                    msg += '</pre>'
+            msg = 'Server got itself in trouble'
+            if 'text/html' in request.headers.get('Accept', ''):
+                if self.debug:
+                    with suppress(Exception):
+                        tb = html_escape(traceback.format_exc())
+                        msg = '<h2>Traceback:</h2>\n<pre>{}</pre>'.format(tb)
+                message = (
+                    "<html><head>"
+                    "<title>500 Internal Server Error</title>"
```

Comment:
`500 Internal Server Error` is reused 3 times, you could put it in a var

Reply:
Actually you could probably render it from `HTTPStatus.INTERNAL_SERVER_ERROR.description` (Ref: https://docs.python.org/3/library/http.html#http-status-codes)

Reply:
`msg_title = http.HTTPStatus.INTERNAL_SERVER_ERROR.description`?